### ISSUE TITLE
Does it support ROS2 foxy?

### ISSUE THREAD

**Issue by YuminosukeSato** [aniskoubaa/rosgpt#7]
Does it support ROS2 foxy?
I would be happy if you could use ROS2's foxy.

**Comment by aniskoubaa** [comment]
Yes, it should support it. I did not use anything specific for ROS2 Humble

**Comment by amirobotic** [comment]
I tested in ROS2 Foxy and it worked. 
This is amazing @aniskoubaa Thanks for all your contributions to the community.
